bin/reth/src/main.rs

```diff
@@ -17,11 +18,25 @@ fn main() {
         unsafe { std::env::set_var("RUST_BACKTRACE", "1") };
     }
 
-    if let Err(err) = Cli::<EthereumChainSpecParser>::parse().run(async move |builder, _| {
-        info!(target: "reth::cli", "Launching node");
-        let handle = builder.node(EthereumNode::default()).launch_with_debug_capabilities().await?;
-        handle.node_exit_future.await
-    }) {
+    if let Err(err) =
+        Cli::<EthereumChainSpecParser, RessArgs>::parse().run(async move |builder, ress_args| {
+            info!(target: "reth::cli", "Launching node");
+            let NodeHandle { node, node_exit_future } =
+                builder.node(EthereumNode::default()).launch_with_debug_capabilities().await?;
+
+            // Install ress subprotocol.
+            let _ress_events = install_ress_subprotocol(
```

Comment:
this is likely problematic because this receiver type is never used and because this is unbounded it will just fill up?